packages/neutrino-preset-node/index.js

```diff
@@ -35,11 +35,13 @@ module.exports = (neutrino) => {
   neutrino.use(compile, {
     include: [neutrino.options.source, neutrino.options.tests],
     babel: {
-      plugins: [require.resolve('babel-plugin-dynamic-import-node')],
+      plugins: [[require.resolve('fast-async'), { spec: true }], require.resolve('babel-plugin-dynamic-import-node')],
```

Comment:
@eliperelman so to confirm, we should add only to entry, right? what if there are multiple entries? what if entry is a function?